README.md

```diff
@@ -97,16 +97,35 @@ https://github.com/pandas-dev/pandas
 Binary installers for the latest released version are available at the [Python
 Package Index (PyPI)](https://pypi.org/project/pandas) and on [Conda](https://docs.conda.io/en/latest/).
 
+For the first option, open a terminal, go to the anaconda environment where you want to install pandas and type the following command:
+
 ```sh
 # conda
 conda install -c conda-forge pandas
 ```
 
+If you want to use pip, you will first have to make sure that it is installed. Type:
+
+```sh
+# check that pip is installed
+pip --version
```

Comment:
I think we should consistently use `python -m pip` in the documentation. It is used in many places but not in 
`doc/source/getting_started/index.rst` and `doc/source/getting_started/install.rst`

Reply:
Hello I am new to github. Could you tell me the steps to do what you ask me? Do I have to commit again on the branch I created and make the change?

Reply:
Yes, you have to commit to the branch you created: make changes, git add, git commit, git push

Reply:
It’s normally good